ikasaneip/module/src/main/java/org/ikasan/module/WiretapTriggerConfiguration.java

```diff
@@ -0,0 +1,66 @@
+package org.ikasan.module;
+
+public class WiretapTriggerConfiguration {
+
+    public String flowName;
+
+    public String componentName;
+
+    public String beforeOrAfter;
+
+    public String timeToLiveInSeconds;
```

Comment:
Its ok to express thettl in sec but current wiretap configuration is defined in minutes i think so make sure relevant conversion takes place